WebRoot/public/extuserprov.jsp

```diff
@@ -147,5 +151,10 @@
 		return !isError;
 	}
 </script>
+<% if (request.getAttribute("errorCode") != null) { %>
+	<script>
+		document.getElementById("ZLoginErrorPanel").style.display = "block";
+	</script>
+<% } %>
```

Comment:
No need of `else` block here. It's already managed here: https://github.com/Zimbra/zm-web-client/blob/854a2da5caef46f5b3f643cdff58ed0163105857/WebRoot/public/extuserprov.jsp#L66